doc/ug/_templates/custom_links_sidebar.html

```diff
@@ -1 +1,2 @@
+<a href="doxygen">Doxygen</a>
 <a href="http://www.espresso-pp.de/Impressum.html">Impressum</a>
```

Comment:
Also I think due to the fact that the website is now hosted on github, the Impressum is obsolete.